client/bindings/wasm/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@iota/iota-client-wasm",
+  "version": "0.5.0-alpha.6",
```

Comment:
Was this ever published somewhere ?

Reply:
Yes. See https://www.npmjs.com/package/@iota/iota-client-wasm

Reply:
This is currently a peer dependency of our identity WASM bindings.

Reply:
Alright, I think we may decide to re-release it just named `client-wasm` to be similar to the client since iota is already in the namespace